packages/app/src/components/inspector/NameInspector.tsx

```diff
@@ -8,10 +9,11 @@ interface Props {
 }
 
 export default memo(function NameInspector({ names, onNameChange }: Props) {
-  const firstName = names[0];
+  const firstName = delimitedPath.basename(names[0]);
```

Comment:
I just realized we're handling `names` and `onNameChange` inconsistently:

- `names` are all full paths, like `["group/nestedGroup/a"]`
- `onNameChange` only gets called with the base name, `"a"`.

I think we should use either full path or base name for both props. `NameInspector` doesn't need to know the full path, cause it doesn't show it, so I would transform `names` into an array of base name, `["a"]`, before passing it in as a prop. It might mean a little duplicated code in the other inspectors, but we can improve that in other ways

Reply:
Oh! It took me a second to understand but I got it, the action uses ids to change the name so i don't need the full name